src/server/internalServer.cpp

```diff
@@ -85,6 +85,16 @@ namespace kiwix {
 namespace
 {
 
+bool ipAvailable(const std::string addr)
+{
+  auto interfaces = kiwix::getNetworkInterfacesIPv4Or6();
+
+  for (const auto& [_, interfaceIps] : interfaces)
+    if ((interfaceIps.addr == addr) || (interfaceIps.addr6 == addr)) return true;
```

Comment:
Move the return statement onto a line of its own and add braces for bodies of both `for` and `if` statement.

Reply:
This was not done.

Reply:
Lost in de-conflicting